controllers/src/test/scala/cromwell/pipeline/controller/UserControllerTest.scala

```diff
@@ -139,8 +132,7 @@ class UserControllerTest
         val accessToken = AccessTokenContent("0")
         val request = UserUpdateRequest(dummyUser.email, dummyUser.firstName, dummyUser.lastName)
 
-        when(userService.updateUser(userId, request))
-          .thenReturn(Future.failed(new RuntimeException("Something wrong.")))
+        when(userService.updateUser(userId, request)).thenReturn(Future.failed(new RuntimeException("Something wrong.")))
```

Comment:
May be 160 maxColumn is too much? it would be better to set it ti ~140-150 ? (I need to scroll horizontally here now, on github page)

Reply:
As I remember, 160 is because Iaroslav showed that some class would place every param on a new line if character length would be less than that.

Reply:
> May be 160 maxColumn is too much? it would be better to set it ti ~140-150 ? (I need to scroll horizontally here now, on github page)

As we discussed this is issue today:
Github change formatting of code for the web page. 
:)